backend/keystore/keystore.go

```diff
@@ -92,9 +93,16 @@ type Keystore interface {
 	// verify.
 	CanVerifyAddress(coin.Coin) (secureOutput bool, optional bool, err error)
 
-	// VerifyAddress outputs the public key at the given configuration for the given coin.
+	// VerifyAddressBTC displays a Bitcoin address for verification.
 	// Please note that this is only supported if the keystore has a secure output channel.
-	VerifyAddress(*signing.Configuration, coin.Coin) error
+	VerifyAddressBTC(
+		accountConfiguration *signing.Configuration,
+		derivation btctypes.Derivation,
+		coin coin.Coin) error
```

Comment:
is there a specific reason we don't directly use `btc.Coin` and `eth.Coin` (below) so we don't have to cast it inside the method?

Reply:
Because of a circular import between the packages :sweat_smile: Maybe we can think another time of if and how to break the cycle.